content/broker/user-guides/_index.nb.md

```diff
@@ -1,46 +1,24 @@
 ---
 title: Brukerveiledninger
 linktitle: Brukerveiledninger
-description: Hvordan komme i gang med Altinn 3 Formidling, for utviklere tjenesteeiere og sluttbrukere?
+description: Hvordan komme i gang med Altinn 3 Formidling
+tags: [Broker, guide]
 toc: true
 weight: 90
 ---
 
 {{<children />}}
 
-<!-- 2024-04-06, Erik: Vurder å gi guides per "value stream stage", 
-                 med egne kapitler for hvert stage, dvs. Onboarding, osv.  
--->
-## Veiledning for utviklere
+På grunn av de ulike distinkte rollene som et foretak kan samhandle med Altinn Formidling, har vi delt inn veiledningene inn i flere deler for kurant rolle:
 
-Se [readme fil på github](https://github.com/Altinn/altinn-broker/blob/main/README.md) for en introduksjon  til Altinn 3 Broker for utviklere (kun på engelsk).
-Her finner du også [Getting started guide for developers](https://github.com/Altinn/altinn-broker/blob/main/docs/get-started.md).
+- Sender/Sender
```

Comment:
Sender/Avsender?

Reply:
Will correct so that it becomes "avsender" consistently in norwegian.